duckduckgo/iOS
# Add widget onboarding flow and onboarding experiment

Task/Issue URL: https://app.asana.com/0/1187352151074490/1194681707917578
Tech Design URL:
CC: @brindy 

**Description**:
Adds a new widget onboarding flow, along with an experiment to determine which onboarding flow is best (default browser, home row, or widget). The experiment should only show for iOS 14 users when the app is in English.

<img width="287" alt="Screenshot 2020-10-19 at 12 59 04" src="https://user-images.githubusercontent.com/1093508/96448147-e07a0c80-120a-11eb-8e6d-8becccb9a6f1.png"> <img width="287" alt="Screenshot 2020-10-19 at 12 57 09" src="https://user-images.githubusercontent.com/1093508/96448177-e8d24780-120a-11eb-86ac-3f435160e053.png">




**Steps to test this PR**:
1. Delete and reinstall the app repeatedly to ensure all variants are bucketed correctly
2. Delete and reinstall the app repeatedly on an device with iOS <14 to ensure it is not part of the experiment (and only sees the home row onboarding)
3. Same as above for when the app is in a language other than English 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
